### DISCUSSION TITLE
What's next?

### DISCUSSION THREAD

**Discussion by ydanneg** [bkbnio/kompendium#203] [General]
Hello.

After some failed attempts to find a working openapi/swagger solution for Ktor I found this project.
And this project works at the minimum level i actually need. Of course a huge amount of work should be done additionally to make it a standard.

I want to say thanks to an author.

I would like to talk about the future of this project. Imho the best way is to create a channel somewhere in Telegram or Discord or Slack.

I have a lot to suggest and discuss.

@unredundant do you have such a channel where we can discuss in a real time?

**Comment by brizzbuzz** [comment]
> I have a lot to suggest and discuss.

Awesome :) Would love to hear them. 

> do you have such a channel where we can discuss in a real time?

No... at the moment I don't have any plans to create an official one.  I feel like GitHub Issues/Discussions/Projects is a pretty good platform, particularly since the community for this project is so small.  

Maybe if this project really picks up traction we will need one, though I still hope that one day Ktor will natively support OpenAPI generation.

**Reply by ydanneg** [reply]
i just got confirmation that Jetbrains is already working on Ktor openapi generation.
The first result will be published at the beginning of summer. Solution will not use any annotations and will be based on code analysis. let's see.
anyway. i will continue using Kompendium and contribute.

thanks

**Reply by brizzbuzz** [reply]
Amazing!  Can't wait to see what they put out.  Hopefully it comes out near the Ktor 2 release... then we can retire Kompendium as a Ktor v1 product.  

But no harm in fixing Kompendium up while we wait :)

**Reply by brizzbuzz** [reply]
I was poking around with their V2 library beta and it seems quite nice.  The route specific plugins is very neat, wondering if they will leverage that in order to avoid annotations
